### PR TITLE
Update 101 readme.

### PR DESCRIPTION
**Pull request by poetw**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 101/README.md [hunk 1]
```diff
@@ -1,29 +1,81 @@
 # 101 Sample
-
-This sample give a quick start to build docker images and use helm chart to deploy a kubernetes pod using the docker image.
+This sample guides you through the full development flow. Follow this README to get familiar with the processes to setup your dev environment and build/deploy/validate sample codes using the Azure Arc VS code extension or command line options.
 
 ## Getting Started
 
-### Prerequisites
+### Recommended Dev Environment Setup
+- Visual Studio Code with [Azure Arc Extension](https://marketplace.visualstudio.com/search?term=azure%20arc&target=VSCode&category=All%20categories&sortBy=Relevance) from VS code marketpalce installed to simplify some development steps
```
Comment (RemindD) [inline 101/README.md:7]: marketplace typo

File: 101/README.md [hunk 2]
```diff
@@ -1,29 +1,81 @@
...
+    for example:
+    ```powershell
+    docker build -t docker.io/johndoe/101sample:latest -f .\101\dotnet\src\Dockerfile .\101\dotnet\src
+    docker push docker.io/johndoe/101sample:latest
+    ```
+
+### Deploying
+Once you have the image ready, you are ready to deploy the image to your K8S cluster.
+1. Make sure your K8S cluster to have sufficient privileges to pull images from it, e.g., you might need to provide imagePullSecrets in the values.yaml of the helm chart, or configure your container registry to allow anonymous pull.
+2. Fill out the variables in 101\101chart\values.yaml. The only mandatory field is the **repository** property under the **image** section, which should be the image repo + tag from the build step.
```
Comment (RemindD) [inline 101/README.md:49]: repository property doesn't need tag. Tag should be put in tag property

**Review by RemindD — APPROVED** [review]

File: 101/README.md (hunk 1 above)
Comment (poetw) [reply 101/README.md:7]: Fixed.

File: 101/README.md (hunk 2 above)
Comment (poetw) [reply 101/README.md:49]: fixed.

**Pull request merged**